cypress/e2e/account/selfEditAccountSettings.feature

```diff
@@ -0,0 +1,34 @@
+@regression_test
+@stable_test
+Feature: Account - Edit e-mail in the account setting page on dBC
+
+    As a user I want to navigation to my account's settings page so that I can see and edit my account details (e. g. email).
+
+    Scenario: I see my e-mail is editable as an internal user, including pre conditions
+        Given I am logged in as a '<user>' at '<namespace>'
+
+        # user changes/updates the e-mail id
+        When I go to my account settings
+        Then I see my email is editable
+        When I change my email to a new email
+        When I enter my current password
+        When I click on the button Save Account Settings
+        Then I see the message successful
+
+        @staging_test
```

Comment:
comment out this as well, otherwise someone will by mistake run this test on staging

Reply:
done!